include/deal.II/lac/utilities.h

```diff
@@ -33,6 +36,67 @@ namespace Utilities
   namespace LinearAlgebra
   {
 
+    /**
+     * Return elements of continuous Givens rotation matrix and norm of the input vector.
```

Comment:
-> Return the elements of a continuous Givens rotation matrix and the norm of the input vector.